cloud_enum.py

```diff
@@ -69,6 +67,8 @@ def parse_arguments():
 
     parser.add_argument('-l', '--logfile', type=str, action='store',
                         help='Will APPEND found items to specified file.')
+    parser.add_argument('-f', '--format', type=str, action='store', default='json', 
+                        help='Format for log file (json or csv, defaults to json)')
```

Comment:
To avoid breaking changes (unless you don't mind), maybe it's worth making this default to `None` and if that's the case then save the output in the current, existing format?

Reply:
Hmmm, good point. I'm not so concerned about breaking what was already a hard to parse log file. But perhaps it's worth adding a "text" option as the default that logs it as plain text, without the ansi stuff. Some might be more familiar with that.

Reply:
Done!